docs/05-10-api.md

```diff
@@ -1,25 +1,62 @@
 # Watcher CR
 
-The Watcher CR configures Kyma Control Plane (KCP) setup and Runtime Watcher on Kyma runtimes. Read the document to see the the parameters the CR consists of. **spec.labelsToWatch** Configures the validation webhook configuration in a Kyma runtime (SKR)
+The [Watcher CR](https://github.com/kyma-project/lifecycle-manager/blob/main/api/v1beta2/watcher_types.go#L121) configures Kyma Control Plane (KCP) setup and Runtime Watcher on Kyma runtimes. Read the document to see the the parameters the CR consists of.
 
-## **spec.labelsToWatch**
+## Runtime-Watcher configuration
 
-The resources you're watching need to have the `operator.kyma-project.io/watched-by` label. The label also informs where to route requests to.
+### **spec.resourceToWatch**
 
-## **spec.resourceToWatch**
+Defines the resources to watch by Runtime Watcher. By default, the **version** value is a wildcard (`*`) so an update is not required after the API version of a kind changes.
 
-Defines the resources to watch by Runtime Watcher. By default, the **version** value is a wildcard (`*`) so an update is not required after the upgrade of the Kyma CRs API version.
+```yaml
+spec:
+  resourceToWatch:
+    group: operator.kyma-project.io
+    version: "*"
+    resource: kymas
+```
 
-## **spec.field**
+### **spec.labelsToWatch**
+> **NOTE:** The resources you're watching need to have the `operator.kyma-project.io/watched-by` label. The label also informs where to route requests to.
 
-<!--TBD-->
+The **spec.labelsToWatch** attribute is used to be able to filter the specified Group/Version/Kind (GVK) in **spec.resourceToWatch**. For example, if the specified GVK is `secrets`, then it is useful to filter them by a specific label, otherwise, the Runtime Watcher would send an event to the KCP for every Create/Read/Update/Delete (CRUD) event of any Secret in the Kyma cluster.
 
-## **spec.serviceInfo**
+```yaml
+spec:
+  labelsToWatch:
+    "operator.kyma-project.io/watched-by": "lifecycle-manager"
+    "example.label.to.watch": "true"
+  resourceToWatch:
+    group: ""
+    version: "v1"
+    resource: secrets
+```
 
-<!--TBD-->
 
-## **spec.gateway**
+### **spec.field**
+It uses either the `spec` or `status` value to define if Runtime Watcher should send an event to KCP if the spec of the specified GVK or the status changes.
 
-<!--TBD-->
+## KCP configuration
+
+### **spec.serviceInfo**
+Specifies the name and Namespace, as well as the port, the incoming events should be routed to.
```

Comment:
```suggestion
Specifies the name and Namespace, as well as the port, the incoming events are routed to.
```
[Trying to avoid "should"](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#terminology).